src/ng/q.js

```diff
@@ -325,14 +364,15 @@ function qFactory(nextTick, exceptionHandler) {
     }
   }
 
-  function scheduleProcessQueue(state) {
+  function scheduleProcessQueue(promise, state) {
     if (state.processScheduled || !state.pending) return;
```

Comment:
Please explain what you mean in a different way. I don't understand this either.